src/app/home-page/recent-item-list/recent-item-list.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RecentItemListComponent } from './recent-item-list.component';
+
+describe('RecentItemListComponent', () => {
+  let component: RecentItemListComponent;
+  let fixture: ComponentFixture<RecentItemListComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ RecentItemListComponent ]
```

Comment:
Tests currently failing, needs provider for newly used `SearchService` mock injected in `RecentItemListComponent` spec. (possibly other services in constructor as well if you add tests for the methods using them)
Also would be good to add some tests here.